### PR TITLE
Add glibcxx_assertion check

### PR DESCRIPTION
**Pull request by badumbatish**
ChangeLog:

	* .github/workflows/ccpp.yml: Add glibcxx_assertion check

### CODE REVIEW

**Comment by badumbatish** [comment]
will try again tmr

**Comment by badumbatish** [comment]
fixed by adding a boolean check of binder.empty()

File: .github/workflows/ccpp.yml [hunk 1]
```diff
@@ -94,6 +94,88 @@ jobs:
...
+    - name: Check for new warnings
+      run: |
+           cd gccrs-build
+           < log grep 'warning: ' | sort > log_warnings
+           if diff -U0 ../.github/glibcxx_ubuntu64b_log_expected_warnings log_warnings; then
+               :
+           else
+               echo 'See <https://github.com/Rust-GCC/gccrs/pull/1026>.'
+               exit 1
+           fi >&2
```
Comment (CohenArthur) [inline .github/workflows/ccpp.yml:155]: so do we need a different expected warnings file? are there more warnings when `GLIBCXX_ASSERTIONS` is defined?

**Review by CohenArthur — APPROVED** [review]
thanks!

File: .github/workflows/ccpp.yml (hunk 1 above)
Comment (badumbatish) [reply .github/workflows/ccpp.yml:155]: the default ubuntu 64 bit has a few more warnings than when we run it with GLIBCXX_ASSERTIONS, thus triggering failure when CI is run. Should we use a different diff at line 150?

**Pull request merged**